gitops/README.md

```diff
@@ -84,7 +84,9 @@ resources:
   pipelines:
     - pipeline: ciPipeline
       source: ci-podinfo
-      trigger: true
+      trigger:
+        stages:
+        - cd_trigger # Stage will only trigger if the build reason isn't pull request
```

Comment:
Is it not possible to trigger using branches?
```
# this is being defined in app-ci pipeline
resources:
  pipelines:
  - pipeline: securitylib   # Name of the pipeline resource
    source: security-lib-ci # Name of the pipeline referenced by the pipeline resource
    trigger: 
      branches:
      - releases/*
      - master
```

Reply:
@phillebaba yeah, but then we can't be flexible around someone using non-default names on their branches. `develop` etc.

Reply:
Ẁhy not? Just change master to develop? This is set in the CD pipeline and not in the template.